syntax = "proto3";

package rpc;

option cc_generic_services = true;

message EchoRequest {
  string sentence = 1;
}

message EchoResponse {
  string result = 1;
}

service EchoService {
  rpc Echo(EchoRequest) returns(EchoResponse);
}